ramls/item.json

```
@@ -176,6 +176,34 @@
             "type": "boolean",
             "description": "Records the fact that the note should only be displayed for staff",
             "default": false
+          },
+          "source": {
+            "type": "object",
+            "description": "The user who added the note",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The id of the user who added the note"
+              },
+              "personal": {
+                "type": "object",
+                "description": "Personal information about the user",
+                "properties": {
+                  "lastName": {
+                    "description": "The user's last name",
+                    "type": "string"
+                  },
+                  "firstName": {
+                    "description": "The user's first name",
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          },
+          "date": {
```

Is this the date when the note was added?

If so, could it have a more explicit name like dateAdded?

---

Actually it is the date when the note is added or updated. As I understood from the Emma's comment from the following story it should work in this way. https://issues.folio.org/browse/MODINV-129

---

Then I suggest the property is named explicitly for that purpose.

```suggestion
          "dateAddedOrUpdated": {
```
```suggestion
          "createdOrUpdatedDate": {
```